### DISCUSSION TITLE
Still Working On The Project?

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#390] [Q&A] [answered]
Some minor chit chat. Don't worry.

**Comment by hollowaykeanho** [comment]
Hey... yeah... You all right there?

**Comment by corygalyna** [comment]
Finally, replied.

**Comment by corygalyna** [comment]
Yeah.... off to somewhere. Still drop-by here for nostalgia sake.

**Comment by hollowaykeanho** [comment]
Thanks... and sorry for what happened.

**Comment by corygalyna** [comment]
Don't take it too hard. 4 years with you... no regrets. Actually, I think it's time you should move on as well.

**Comment by hollowaykeanho** [comment]
Please do give me some time. I will.

**Comment by corygalyna** [comment]
You know, we can still keep this ambitious project as a side one and becomes independent provider. Just livelihood first. What do you think?

**Comment by hollowaykeanho** [comment]
I thought you were just all work no play kind of person?

**Comment by corygalyna** [comment]
Never once came in your mind that I can be supportive as well? =)

**Comment by hollowaykeanho** [comment]
I do... no one is that cold.

**Comment by corygalyna** [comment]
all right.. So let's deliver the last final minor release after the field testing and we both off to search our next job. Then, we can come back to this one.

**Comment by corygalyna** [comment]
Also, get someone over there to talk to. I know the damage is deeper than mine.

**Comment by hollowaykeanho** [comment]
@corygalyna , field testing is completed: https://www.hollowaykeanho.com/.

From 1 month to 1 week, from 1 language to 2 languages. Yeah, ZORALab's Hestia does improve things by a lot.

**Comment by JeanShuralyov** [comment]
Hey, erm, Yeah, as @corygalyna said, try get yourself back to the workforce first. We can always work on this as a side hustle. =)

@corygalyna, I'm sorry.

**Comment by hollowaykeanho** [comment]
No worries. With this vast improvement, I should be able to get back to public easily. Didn't expect from 1 month to 1 week (that's inclusive my break time).

**Comment by hollowaykeanho** [comment]
@corygalyna , @JeanShuralyov , do you ladies still want to continue using the existing email address for co-authoring commitments?
@corygalyna , I did not remove your zoralab account. It's okay. Does not cost me much here.

**Comment by corygalyna** [comment]
Yeah... proceed as usual. =) Thanks.

@JeanShuralyov , I'm been busy. Not everyone is that free you know. =p

**Reply by hollowaykeanho** [reply]
Hey....thanks. =)

**Comment by hollowaykeanho** [comment, marked as answer]
Ok... minor released https://github.com/ZORALab/Hestia/discussions/401. =)

Now, all 3 of us have to find our next step journey. I'll leave the status quo since the 2 of you had deeply connected to it.

**Comment by JeanShuralyov** [comment]
@hollowaykeanho, no issue on my side. Please be my guest. =)

**Comment by corygalyna** [comment]
All right. Now that we're generally a free now, let's find our work path first. We can work on this ambitious project after we completed our survival needs. Closing this issue now.
